src/universal/modules/menu/ducks/menuDuck.js

```diff
@@ -8,22 +8,27 @@ const initialState = {
 };
 
 export default function reducer(state = initialState, action = {}) {
-  const {type, payload: {menuKey}} = action;
```

Comment:
if it's performance you're going for, if/else statements are a TON faster than switch/case. No idea why...
Alternatively we could kick it off with a `if (!action.type.startsWith('/menu') return state`